client/one-time-payment/html/src/custom/sandboxedIframe/src/paypal-iframe/integration.js

```diff
@@ -105,9 +76,32 @@ function setupIframeOriginDisplay() {
 
 async function setupPayPalButton(sdkInstance) {
   pageState.paymentSession = sdkInstance.createPayPalOneTimePaymentSession({
-    onApprove,
-    onCancel,
-    onError,
+    onApprove: async (data) => {
```

Comment:
Is there a strong reason to do these functions inline as opposed to declaring them separately like before? I have no strong opinions but just curious to know why we're choosing this pattern 🙂

Reply:
I could go either way with this. In the other examples I did this pattern:
```js
sdkInstance.createPayPalOneTimePaymentSession({
  async onApprove(data) {
  }
});
```

Reply:
I don't have strong opinions on this either, just responding to the feedback that it would be more consistent to match the pattern in other examples. I'm down to update this however we'd like though, let me know what changes I should make!

Reply:
Yeah nothing super pressing here we can just follow the other examples and keep it like this 🙂